Kraft menu crashes with nil pointer dereference

### Describe the bug

Running `kraft menu` crashes with a nil pointer dereference.

### Steps to reproduce

Clone the [app-helloworld](https://github.com/unikraft/app-helloworld) repository and run `kraft menu`.

### Expected behavior

The `menuconfig` to pop up.

### Which architectures were you using or does this bug affect?

_No response_

### Which operating system were you using or does this bug affect?

_No response_

### Relevant log output

```shell
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x21b6735]

goroutine 1 [running]:
kraftkit.sh/cmd/kraft/menu.(*Menu).pull(0xc0000be800, {0x2a1d600, 0xc00071a8d0}, {0x2a39158, 0xc00050f400}, {0xc000058064, 0x22}, 0x0, 0x23)
	/home/kero/unikraft-work/kraftkit/cmd/kraft/menu/menu.go:216 +0x975
kraftkit.sh/cmd/kraft/menu.(*Menu).Run(0xc0000be800, 0x0?, {0x0?, 0x0?, 0x0?})
	/home/kero/unikraft-work/kraftkit/cmd/kraft/menu/menu.go:386 +0x1ec
kraftkit.sh/cmdfactory.AttributeFlags.bind.func3(0xc00048c100?, {0x39ce8c0, 0x0, 0x0})
	/home/kero/unikraft-work/kraftkit/cmdfactory/builder.go:543 +0x1b6
github.com/spf13/cobra.(*Command).execute(0xc000005500, {0x39ce8c0, 0x0, 0x0})
	/home/kero/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000004900)
	/home/kero/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	/home/kero/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/home/kero/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:985
kraftkit.sh/cmdfactory.Main({0x2a1d600?, 0xc0004e34d0}, 0xc000004900)
	/home/kero/unikraft-work/kraftkit/cmdfactory/builder.go:183 +0x74
main.main()
	/home/kero/unikraft-work/kraftkit/cmd/kraft/kraft.go:163 +0x5e5
```

--- a/cmd/kraft/menu/menu.go
+++ b/cmd/kraft/menu/menu.go
@@ -213,7 +213,7 @@ func (opts *Menu) pull(ctx context.Context, project app.Application, workdir str
 		}
 	}
 
-	if opts.project.Template().Name() != "" {
+	if opts.project.Template() != nil && opts.project.Template().Name() != "" {
 		templateWorkdir, err := unikraft.PlaceComponent(workdir, opts.project.Template().Type(), opts.project.Template().Name())
 		if err != nil {
 			return err

